db/liquibase/changesets/20180915_collection.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="avram" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="avram" author="ansingh" id="collection11">
+        <createTable tableName="COLLECTION">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="VARCHAR(254)">
```

Comment:
Nit: `VARCHAR(255)` is a common convention, I don't know why all our other stuff uses 254. :)

Reply:
Also, I see no reason to limit this to a couple hundred characters. I think databases these days are pretty good at optimizing storage so there shouldn't be a penalty for increasing the width of `varchar` fields (though we should check this about postgres; most of my experience is with oracle).

I just did a little reading here: https://www.postgresql.org/docs/10/static/datatype-character.html and here: https://stackoverflow.com/a/4849030/244191. TL;DR: all character types perform reasonably similar. Max size of a column is about 1G. Also postgres adds an extension to `varchar` that allows you to not specify a length as well as a `text` type that is the same as an unbounded `varchar`.

I propose we use something like `varchar(1000)` to allow for a long name and protect us from excessive data storage requirements.